static/js/src/advantage/react/components/Subscriptions/SubscriptionDetails/DetailsContent/DetailsContent.tsx

```diff
@@ -93,22 +100,39 @@ const DetailsContent = ({ selectedId, setHasUnsavedChanges }: Props) => {
   const tokenBlock = token?.contract_token ? (
     <div className="u-sv4 u-no-margin--bottom p-code-snippet">
       <pre
-        className="p-code-snippet__block--icon is-url"
+        className="p-code-snippet__block--icon is-url p-code-snippet__block--hover"
         data-test="contract-token"
         onClick={() => {
```

Comment:
Would it be better to use a function for `onClick` instead of inline for better readability? (optional)

Reply:
Done.